bump.lua

```diff
@@ -194,8 +227,7 @@ local function rect_detectCollision(x1,y1,w1,h1, x2,y2,w2,h2, goalX, goalY)
     move      = {x = dx, y = dy},
     normal    = {x = nx, y = ny},
     touch     = {x = tx, y = ty},
```

Comment:
Stop returning itemRect and otherRect, because I found them to be simply unecessary in my own use. If a user needs them, they can simply call `World:getRect()`, if they don't need them, then we've saved some garbage right here.

We cache distance here, because it will often always be needed for `sortByTiAndDistance`. And when it *is* needed, it saves time to calculate it once here, rather than every time `table.sort()` needs to compare two objects.

Reply:
Although it occurs to me now that this is not "distance", but rather "squared distance". I wonder if it's worthwhile to math.sqrt() this... hmmm